docs/REST/TEC/V1/endpoints.md

```diff
@@ -94,6 +94,34 @@ https://yoursite.com/wp-json/tec/v1
 - **Operation IDs**: `getVenue` (GET), `updateVenue` (PUT), `deleteVenue` (DELETE)
 - [Full Documentation](endpoints/venues.md#single-entity-endpoint)
 
+### Tickets (Event Tickets Plugin)
```

Comment:
Should the tickets data be included here? Shouldn't it only be included in Event Tickets?